javascript/javascript2/week2/movies.js

```diff
@@ -0,0 +1,2628 @@
+        year: 1982,
+        rating: 7,
+        votes: 41135,
+        running_times: 7440,
+    },
+    {
+        title: "An Unfinished Life",
+        year: 2005,
+        rating: 7,
+        votes: 25784,
+        running_times: 6480,
+    },
+    {
+        title: "Anaconda",
+        year: 1997,
+        rating: 4.7,
+        votes: 80526,
+        running_times: 5340,
+    },
+    {
+        title: "Anacondas: The Hunt for the Blood Orchid",
+        year: 2004,
+        rating: 4.6,
+        votes: 24190,
+        running_times: 5820,
+    },
+    {
+        title: "Analyze That",
+        year: 2002,
+        rating: 5.9,
+        votes: 74001,
+        running_times: 5760,
+    },
+    {
+        title: "Analyze This",
+        year: 1999,
+        rating: 6.7,
+        votes: 131399,
+        running_times: 6540,
+    },
+    {
+        title: "Anastasia",
+        year: 1997,
+        rating: 7.1,
+        votes: 95743,
+        running_times: 5640,
+    },
+    {
+        title: "Anatomy of a Murder",
+        year: 1959,
+        rating: 8.1,
+        votes: 50250,
+        running_times: 9600,
+    },
+    {
+        title: "Anchorman 2: The Legend Continues",
+        year: 2013,
+        rating: 6.3,
+        votes: 143995,
+        running_times: 8580,
+    },
+    {
+        title: "Anchorman: The Legend of Ron Burgundy",
+        year: 2004,
+        rating: 7.2,
+        votes: 288977,
+        running_times: 6240,
+    },
+    {
+        title: "And Now for Something Completely Different",
+        year: 1971,
+        rating: 7.6,
+        votes: 25816,
+        running_times: 5280,
+    },
+    {
+        title: "And So It Goes",
+        year: 2014,
+        rating: 5.8,
+        votes: 10471,
+        running_times: 5640,
+    },
+    {
+        title: "And Soon the Darkness",
+        year: 2010,
+        rating: 5.2,
+        votes: 14637,
+        running_times: 5460,
+    },
+    {
+        title: "And Then There Were None",
+        year: 1945,
+        rating: 7.6,
+        votes: 10271,
+        running_times: 5820,
+    },
+    {
+        title: "Angel Eyes",
+        year: 2001,
+        rating: 5.6,
+        votes: 20391,
+        running_times: 6120,
+    },
+    {
+        title: "Angel Heart",
+        year: 1987,
+        rating: 7.3,
+        votes: 68210,
+        running_times: 6780,
+    },
+    {
+        title: "Angela's Ashes",
+        year: 1999,
+        rating: 7.3,
+        votes: 18063,
+        running_times: 8700,
+    },
+    {
+        title: "Angels & Demons",
+        year: 2009,
+        rating: 6.7,
+        votes: 235481,
+        running_times: 8760,
+    },
+    {
+        title: "Angels in the Outfield",
+        year: 1994,
+        rating: 6.1,
+        votes: 24230,
+        running_times: 6120,
+    },
+    {
+        title: "Angels with Dirty Faces",
+        year: 1938,
+        rating: 8,
+        votes: 17119,
+        running_times: 5820,
+    },
+];
+
```

Comment:
It will be great if you can put this json data to a separate file, and just import it wherever you need